ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -238,6 +245,44 @@ def install_internal_cluster(self):
         logger.info("Sleep 10 second after click on 'create storage cluster'")
         time.sleep(10)
 
+    def install_lso_cluster(self):
+        """
+        Install LSO cluster on AWS attached devices
+
+        """
+        logger.info("Click Internal - Attached Devices")
+        self.do_click(locator=self.dep_loc["attached_devices_mode"])
+
+        # All nodes will be added by default. TODO: add node selection option
+        logger.info("Go to Step 2")
+        self.do_click(locator=self.dep_loc["submit_button"])
+
+        logger.info("Enter volume set name")
+        self.do_send_keys(self.dep_loc["volume_set_name"], text="localblock")
+
+        logger.info("Go to Step 3")
+        self.do_click(locator=self.dep_loc["submit_button"])
+
+        logger.info("Confirm storage class creation")
+        self.do_click(locator=self.dep_loc["yes_button"])
+
+        logger.info("Sleep for 60 seconds to wait for nodes to be discovered")
+        time.sleep(60)
+
+        logger.info("Go to Step 4")
+        self.do_click(locator=self.dep_loc["submit_button"])
+
+        # Encryption is disabled by default. TODO: add encryption
+
```

Comment:
@ebondare Can use this configure_encryption function on lso?
https://github.com/red-hat-storage/ocs-ci/blob/afd9e58b16b8729f9fe04c76a5e08b0d285a4403/ocs_ci/ocs/ui/deployment_ui.py#L241